### PR TITLE
Make created_at value follows RFC3339 time format.

### PR DESCRIPTION
**Pull request by jiyongjung0**
Make created_at value of default_version in UploadPiplineResponse follow RFC3339 time format.

It returned `created_at` value as a json dictionary, for example
`"created_at":{"seconds":1576824474}`, and broke SDK parser.  `jsonpb`
will be used to take care of all json marshaling including time format
conversion.

See https://github.com/kubeflow/pipelines/issues/2764 for the detail.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2876)

### CODE REVIEW

File: backend/src/apiserver/server/pipeline_upload_server_test.go [hunk 1]
```diff
@@ -51,8 +52,10 @@ func TestUploadPipeline_YAML(t *testing.T) {
 	handler := http.HandlerFunc(server.UploadPipeline)
 	handler.ServeHTTP(rr, req)
 	assert.Equal(t, 200, rr.Code)
-	// Verify time format is RFC3339
-	assert.Contains(t, rr.Body.String(), `"created_at":"1970-01-01T00:00:01Z"`)
+	// Verify time format is RFC3339. There should be two created_at,
+	// one for pipeline and one for the default version. It means that we should
+	// get 3 if we split the output with the expected expression.
+	assert.Equal(t, 3, len(strings.Split(rr.Body.String(), `"created_at":"1970-01-01T00:00:01Z"`)))
```
Comment (Ark-kun) [inline backend/src/apiserver/server/pipeline_upload_server_test.go]: This looks pretty hacky. Is it possible to deserialize and check the particular fields?

Comment (jiyongjung0) [reply backend/src/apiserver/server/pipeline_upload_server_test.go]: I agree with you. I've added deserialization. Thank you for the suggestion! :smile:

**Pull request merged**